scripts/adapters.dockerfile

```diff
@@ -17,8 +17,22 @@ FROM $image
 
 COPY scripts/setup-helper-functions.sh /
 COPY scripts/setup-adapters.sh /
-RUN mkdir build && ( cd build &&  source /opt/rh/gcc-toolset-12/enable && \
-    bash /setup-adapters.sh ) && rm -rf build && dnf remove -y conda && dnf clean all
+COPY scripts/setup-centos9.sh /
```

Comment:
This file should already be present, because this image is based on the CentOS image (and I don't think it is deleted from that image). Should we re-copy it or just assume it is present? I went with copying it again since we are re-copying `setup-helper-functions.sh` above.